src/server/NetworkingServer.cpp

```diff
@@ -150,6 +165,8 @@ void NetworkingServer::ApiServer() {
                                 std::cout << "TCP client disconnected. Removing client. " << std::endl;
                                 DeleteClient(&client, selector);
                                 break;
+                            default:
```

Comment:
were you getting a warning or something? this default is pretty pointless. at least we should put a `cout` saying this happened

Reply:
Yeah I was getting a warning, yeah doing some error logging here would be good. Will add it.

Reply:
Fixed